ecl/hqlcpp/hqlresource.cpp

```diff
@@ -602,7 +602,7 @@ class EclChildSplitPointLocator : public EclHoistLocator
             }
         case no_attr_expr:
             //Ignore internal tracking attributes e.g., _selectors_Atom
-            if (!isInternalAttributeName(expr->queryName()))
+            if (!isInternalAttributeName(expr->queryName()) || expr->queryName() == _loopFirst_Atom)
```

Comment:
This prevented expressions in loopfirst attributes from being executed outside the loop activity.